enclave/core/sgx/exit.S

```diff
@@ -100,6 +104,9 @@ oe_asm_exit:
     // simulate-flag-check.
     lfence
 
+    // Clear %r9 which was being used to maintain td pointer
```

Comment:
I think you have tried to match comment with code, right?

Reply:
Oh good catch, I forgot to change this. Thanks